### ISSUE TITLE
getPortName returns incorrect value

### ISSUE THREAD

**Issue by aolsenjazz** [justinlatimer/node-midi#199]
getPortName returns the incorrect name for the Impulse25 MIDI Controller. Other clients (Ableton, MIDI Monitor) see the device's name as 'Impulse' whereas getPortName return 'Impulse  Impulse '.

Code to reproduce:
````javascript
const midi = require("midi");

const input = new midi.Input();
const portName = input.getPortName(0).split(" ").join("_"); // replace spaces with underscores for visibility

console.log(portName);
// logs Impulse__Impulse_
````

Environment deets:
OSX 11.2
Core MIDI
Node 14.4.0
Please let me know what more info I can provide.

Will investigate this further but I've little-to-no experience with native node libraries :). Thanks for writing and making available such a great library.

**Comment by justinlatimer** [comment]
Looks like the name of the port is determined in RtMidi [here](https://github.com/justinlatimer/rtmidi/blob/master/RtMidi.cpp#L1225). Not sure why it concatenates the name like that?

**Comment by aolsenjazz** [comment]
Thanks for the quick reply. It looks like rtmidi is giving a weird name because the Impulse25 includes the Device name in its Endpoint names (which I suppose isn't? conventional). If you'd like, I can open a PR to check for existence of Device name in Endpoint names and return a more appropriate name.

**Comment by aolsenjazz** [comment]
The device I'm using has a leading whitespace in its endpoint name, messing up that concatenation logic.

Fixed with a [CFStringTrim](https://github.com/thestk/rtmidi/pull/248/files#diff-c9de391d9ec66966051ffa94966bec5f66ad82c231100c1be2c28128b8d86d67R1120).

Opened [PR#248](https://github.com/thestk/rtmidi/pull/248) in rtmidi. Will update if merged.

**Comment by justinlatimer** [comment]
Nice one! If you don't get traction with upstreaming to RtMidi in a few weeks, we might be able to just put in the fork node-midi uses directly.

**Comment by aolsenjazz** [comment]
Excellent. Will open a PR in due time if that's the case. Thanks again!

**Comment by aolsenjazz** [comment]
Merged in rtmidi

**Comment by justinlatimer** [comment]
Updated RtMidi in 7c659437963c311bc62d94898ad8867e220a8899, published to npm in `v2.0.0`
